llama_stack/providers/remote/inference/passthrough/passthrough.py

```diff
@@ -135,10 +145,39 @@ async def chat_completion(
             "logprobs": logprobs,
         }
 
-        params = {key: value for key, value in params.items() if value is not None}
-
         # only pass through the not None params
-        return client.inference.chat_completion(**params)
+        request_params = {key: value for key, value in request_params.items() if value is not None}
+
+        # cast everything to json dict
+        json_params = self.cast_value_to_json_dict(request_params)
+
+        if stream:
+            return self._stream_chat_completion(json_params)
+        else:
+            return await self._nonstream_chat_completion(json_params)
+
+    async def _nonstream_chat_completion(self, json_params: Dict[str, Any]) -> ChatCompletionResponse:
+        client = self._get_client()
+        response = await client.inference.chat_completion(**json_params)
+
+        response = response.to_dict()
+
+        # temporary hack to remove the metrics from the response
+        response["metrics"] = []
```

Comment:
What's the error message on keeping metrics? Maybe @dineshyv could have some insights.

Reply:
@SLR722 can you explain why you have this and document this please?